SMLHelper/Patchers/LanguagePatcher.cs

```diff
@@ -105,41 +101,48 @@ private static void ReadOverrideCustomLines()
 
                 Dictionary<string, string> originalLines = originalCustomLines[modName];
 
-                int overridesApplied = 0;
-                for (int lineIndex = 0; lineIndex < languageLines.Length; lineIndex++)
-                {
-                    string line = languageLines[lineIndex];
-                    if (string.IsNullOrEmpty(line))
-                        continue; // Skip empty lines
+                int overridesApplied = ExtractOverrideLines(modName, languageLines, originalLines);
 
-                    string[] split = line.Split(new[] { KeyValueSeparator }, StringSplitOptions.RemoveEmptyEntries);
+                Logger.Log($"Applied {overridesApplied} language overrides to mod {modName}.", LogLevel.Info);
+            }
+        }
 
-                    if (split.Length != 2)
-                    {
-                        Logger.Log($"Line '{lineIndex}' in language override file for '{modName}' was not correctly formatted.", LogLevel.Warn);
-                        continue; // Not correctly formatter
-                    }
+        internal static int ExtractOverrideLines(string modName, string[] languageLines, Dictionary<string, string> originalLines)
+        {
+            int overridesApplied = 0;
+            for (int lineIndex = 0; lineIndex < languageLines.Length; lineIndex++)
+            {
+                string line = languageLines[lineIndex];
+                if (string.IsNullOrEmpty(line))
+                    continue; // Skip empty lines
 
-                    string key = split[0];
+                string[] split = line.Split(new[] { KeyValueSeparator }, 2, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
For the sake of documentation, replace the literal `2` in all calls into `string.Split` and `split.Length` with a reference to a new constant.
```CSharp 
const int SegmentCount = 2;
```

Good call using [Split(String[], Int32, StringSplitOptions)](https://docs.microsoft.com/en-us/dotnet/api/system.string.split?view=netframework-4.8#System_String_Split_System_String___System_Int32_System_StringSplitOptions_) to avoid the issues of `:` characters in the value text.

Reply:
I'm not sure how to respond do this. Why would we need to define a constant _integer_ value which is only used in 1 or 2 places?

Reply:
Using a well named constant can make it effortless to infer the _meaning_ and _intent_ of the literal value its substituting.  
That makes for clean, self-documenting code that is much more easily understood by others.

Like it or not, SMLHelper is an open source project and we should be making every effort to make it easy for other people to jump in, understand what they are looking at, and be able to contribute.